content/scout/integrations/environment/sysdig.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Integrate with Docker Scout using Sysdig
+description: scout, sysdig, integration, image analysis, environments, supply chain
+keywords: Integrate your runtime environments with Docker Scout using Sysdig
+---
+
+{{< include "scout-early-access.md" >}}
+
+The Sysdig integration enables Docker Scout to automatically detect the images
+you're using for your running workloads. Activating this integration gives you
+real-time insights about your security posture, and lets you compare your
+builds with what's running in production.
+
+## How it works
+
+The Sysdig Agent captures which images are being used to run your container
+workloads. Docker Scout integrates with the Sysdig API to discover the images
+in your cluster. Each Sysdig integration maps to an environment. When an image
+is detected in the cluster, that image gets assigned to the associated
+environment.
+
+The prerequisites for integrating Docker Scout with Sysdig are:
+
+- You must install the Sysdig Agent in the cluster that you want to integrate.
+  For more information, see [Sysdig docs](https://docs.sysdig.com/en/docs/installation/).
```

Comment:
I don't think this is correct. They need Risk Spotlight Integration API token from Sysdig. The link to that is available on scout.docker.com.

Reply:
Hmm. Without any agent installed, how does Sysdig get information about deployments? cc @cdupuis

Reply:
May be in wrong here but I was just comparing what was there on the UI. But cd can through some light here.